### ISSUE TITLE
Cannot import v2 of the module

### ISSUE THREAD

**Issue by pcriv** [carlocayos/go-cod#18]
When trying to import v2 I get the following message:

```
require github.com/carlocayos/go-cod: version "v2.0.0" invalid: should be v0 or v1, not v2
```

**Comment by carlocayos** [comment]
Did a major version update which was causing the issue. I've just added a fix in release `v2.1.0`.

Try creating a new project and import version 2 `v2`.

```
import "github.com/carlocayos/go-cod/v2"
```

When you download the go mod dependencies, your `go.mod` file should be `v2.1.0`.

```
require github.com/carlocayos/go-cod/v2 v2.1.0
```

**Comment by pcriv** [comment]
Thanks for the quick fix! 💯 

If there are things I could help with, let me know! I would really like to contribute to this project :)
